interface/client/lib/helpers/helperFunctions.js

```diff
@@ -50,6 +50,21 @@ Helpers.formatUrl = function(url){
 };
 
 /**
+Sanatizes URLs to prevent phishing and XSS attacks
+
+@method sanatizeUrl
+@param {String} url
+**/
+Helpers.sanatizeUrl = function(url){    
+    // add http:// if no protocol is present
+    if(_.isString(url)) {
+        url = url.replace(/^(javascript:)?(data:)?\/\//i, 'http:');
+    }
+
+    return url;
```

Comment:
Sorry, I missed the `/i` . But javascript can still be broken up with tab or %0A. I'd rather see: 

1. Robust URL parsing, either using DOM anchor element to parse URL or a proper URL parsing library. 
2. Whitelist, only allowing the protocols we want to allow, instead of blacklisting a few.